.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sd
+    docker:
+      - image: msheiny/pressfreedomci:latest
+        environment:
+          FPF_CI: true
+          CI_SD_ENV: staging
+          CI_AWS_TYPE: t2.small
+          FPF_GRSEC: false
+          TEST_REPORTS: /root/sd
+    steps:
+      # This is really dumb that Im doing this, basically
```

Comment:
> Securedrop kept getting cloned to ~/sd/securedrop

That doesn't seem right. According to the reference [here](https://circleci.com/docs/2.0/configuration-reference/#checkout): it should be the case that that step will fail. Of course, as I'm discovering, this documentation sucks and doesn't make sense. I mean if path defaults to working_directory, and working_directory is created if it doesn't exist, then checkout without a path argument should always fail.

Just follow your heart on this one I guess :heart: